assets/js/blocks/product-categories/frontend.js

```diff
@@ -1,34 +1,24 @@
-/**
- * External dependencies
- */
-import { render } from 'react-dom';
-
 /**
  * Internal dependencies
  */
 import Block from './block.js';
 import getCategories from './get-categories';
+import renderFrontend from '../../utils/render-frontend.js';
 
-const containers = document.querySelectorAll(
-	'.wp-block-woocommerce-product-categories'
-);
+const selector = '.wp-block-woocommerce-product-categories';
```

Comment:
Declaring this variable here is unnecessary, can just reference the selector directly in the `renderFrontend` call.

Reply:
Changed in 454c5d708166edd9c32d37586b95b23a9e24095a.